docs/improving-gci-experience.md

```diff
@@ -0,0 +1,377 @@
+in the community. Thus being in the top-10 is a requirement, but the
+student with the most completed tasks won't necessarily win the grand-prize.
+
+## How do I make sure none of my contributions are forgotten?
+
+If you are worried that some of your contributions may not be taken into
+account when choosing the winners, you can create a document which lists
+the achievements that were not GCI tasks.
+
+You can see an example format for this document
+[here](https://paper.dropbox.com/doc/GCI-achievements-example-CT1htUQQnkFkZE1k27ql2).
+If you create your achievement document, you can add it to [this
+list](https://docs.google.com/spreadsheets/d/1eF2xi5_ZrpURH5N_VV99C8o-deLh-A5YjNwWLc4opik/edit?usp=sharing).
+
+**Note**: This is not a requirement for the GCI students. We simply want to
+give students a simple way of showcasing all of their achievements. We will
+still review Zulip pull requests and issues, and Zulip threads in addition
+to the GCI tasks they've completed.
+
+## Why should I claim a task before working on it?
+
+The [GCI
+rules](https://developers.google.com/open-source/gci/resources/contest-rules)
+require you to claim a task before you start working on it. It's recommended
+since students can't see the number of instances available and it may happen
+that someone else claims the last instance before you can claim it. Your work
+won't then be applicable for review.
+
+The workflow for completing GCI tasks:
+
+* Student claims the task.
+
+* Student works on the tasks and makes sure it fulfills all the requirements.
+
+* Before submitting the task, they check that the task passes all the tests
+(including Travis tests and manual testing of the feature).
+
+* Student submits the task in the GCI app and waits for a review - they
+should focus on other endeavors while waiting.
+
+* The task is going through the review process, with refactoring and
+improvements required by mentors.
+
+* The task is accepted when a mentor assesses it as fulfilling the task
+requirements - this doesn't necessarily mean the pull request will be merged
+at the same time.
+
+## How does the review process work?
+
+It's good to have more than one mentor look at your contribution, since they
+can focus on different aspects of it. The main aspects we focus on with the
+GCI tasks:
+
+* General task requirements - the files are named properly and created in the
+right folders, the commit messages are following both the specific task
+description and [Zulip commit discipline
+doc](http://zulip.readthedocs.io/en/latest/version-control.html#commit-discipline)
+etc.
+
+* Task functionality - the submission implements the required functionality
+appropriately.
+
+* Code quality and style (as in [Zulip code style
+  doc](http://zulip.readthedocs.io/en/latest/code-style.html)).
+
+* Passing the test suite.
+
+* Passing manual tests.
+
+* Files are properly rendering (especially Markdown files).
```

Comment:
Maybe "Documentation (e.g. markdown files) renders properly and is nicely readable"